src/software/jetson_nano/gpio.cpp

```diff
@@ -0,0 +1,96 @@
+#include "software/jetson_nano/gpio.h"
+
+#include <fcntl.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "software/logger/logger.h"
+
+GPIO::GPIO(std::string gpio_number, GpioDirection initial_direction,
+           GpioState initial_state)
```

Comment:
seems like there's no way to change the direction so it's not really an initial direction

Reply:
true, changed to direction

Reply:
didn't change argument name here

Reply:
changed it in the header, my bad

changed in the cpp as well